setup.cfg

```diff
@@ -45,7 +46,3 @@ install_requires =
     # pyqubo==1.4.0
 packages = dwaveoceansdk
 python_requires = >=3.8
-
-[options.extras_require]
-all =
```

Comment:
I would leave `all` in with nothing. Because otherwise any `pip install dwave-ocean-sdk[all]` out there will break.

Reply:
Ah, good point.

Reply:
Seems like only a warning is produced when calling an extra that doesn't exist though, so probably no big harm in not having it. It's there as an empty extra now anyway.

Reply:
Ah, it used to be an error, but looks like they changed it to a warning in [63.4.2](https://setuptools.pypa.io/en/stable/history.html#v63-4-2).